paasta_tools/marathon_tools.py

```diff
@@ -911,6 +915,11 @@ def _namespaced_get_classic_service_information_for_nerve(name, namespace, soa_d
     port_file = os.path.join(soa_dir, name, 'port')
     nerve_dict['port'] = service_configuration_lib.read_port(port_file)
     nerve_name = compose_job_id(name, namespace)
+    try:
+        with open(_get_classic_service_puppet_file(name)) as extras:
+            nerve_dict.update(json.load(extras))
+    except (ValueError,IOError):
+        pass
```

Comment:
Yes, great idea. I'll look for docs related to the puppet_services.d directory too, to mention this new behaviour.